L1Trigger/TrackFindingTracklet/interface/Settings.h

```diff
@@ -448,15 +450,15 @@ namespace trklet {
     //have the factor if 2
     double krprojshiftdisk() const { return 2 * kr(); }
 
-    double benddecode(unsigned int ibend, unsigned int layerdisk, bool isPSmodule) const {
```

Comment:
Why have you changed "unsigned" to "int" in the arguments of function Settings::benddecode()?

Reply:
This is fixed.